EIPS/eip-action_standard.md

```diff
@@ -0,0 +1,301 @@
+
+- Aggregator services can analyze action events to determine likely state contracts for a given sender/receiver
+- Sender/receiver contracts can require a specific state contract
+- Sender/receiver contracts can allow any state contract, but set a default. This is important for NFTs that change their render based on state. This default can also be configurable by the token holder.
+- State contracts can be bridges to state contracts on another chain, allowing for L1-verification, L2-storage usage pattern (validate action with layer-1 assets, save on l2 where storage is cheaper).
+
+### Extensions
+
+#### Interactive
+
+Some contracts may have custom user interfaces that facilitate interaction.
+
+```solidity
+pragma solidity ^0.8.0;
+
+interface IERCxxxxInteractive {
+    function interfaceURI(bytes4 _action) external view returns (string);
+}
+```
+
+#### Action Proxies
+
+Action proxies can be used to support backwards compatibility with non-upgradeable contracts, and potentially for cross-chain action bridging.
+
+They can be implemented using a modified version of [ERC-1820](https://eips.ethereum.org/EIPS/eip-1820#erc-1820-registry-smart-contract) that allows [ERC-173](https://eips.ethereum.org/EIPS/eip-173) contract owners to call `setManager()`.
+
+#### Controllable
+
+Users of this standard may want to allow trusted contracts to control the action process to provide security guarantees, and support action bridging. Controllers step through the action chain, calling each contract individually in sequence.
+
+Contracts that support Controllers SHOULD ignore require/revert statements related to action verification, and MUST NOT pass the action to the next contract in the chain.
+
+```solidity
+pragma solidity ^0.8.0;
+
+interface IControllable {
+    function approveController(address sender, string memory action)
+        external
+        returns (bool);
+
+    function revokeController(address sender, string memory action)
+        external
+        returns (bool);
+
+    function isApprovedController(address sender, string memory action)
+        external
+        view
+        returns (bool);
+}
+```
+
+## Rationale
+
+There are many proposed uses for interactions with and between tokenized assets. Projects that are developing or have already developed such features include fully on-chain games like [Realms' cross-collection Quests](https://docs.bibliothecadao.xyz/lootverse-master-scroll/) and the fighting game [wrasslers](https://wrasslers.com/), and partially on-chain games like [Worldwide Webb](https://webb.game/) and [Axie Infinity](https://axieinfinity.com/). It is critical in each of these cases that users are able to commit actions on and across tokenized assets. Regardless of the nature of these actions, the ecosystem will be stronger if we have a standardized interface that allows for asset-defined action handling, open interaction systems, and cross-functional bridges.
```

Comment:
This section reads more like it belongs in Motivation.

The Rationale section explains the reasoning behind choices made inside this EIP, where the Motivation section explains why the EIP itself needs to exist.